apps/docs/layouts/guides/index.tsx

```diff
@@ -1,17 +1,15 @@
 import { MDXProvider } from '@mdx-js/react'
 import { NextSeo } from 'next-seo'
 import Head from 'next/head'
-import Link from 'next/link'
```

Comment:
possible mistake. @ivasilov just double check.

Reply:
Checked manually, the Link is not used anywhere in the page. It would'be broken the build.